### PR TITLE
Add .vsconfig files in root and beside slns

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

File: .vsconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.7.2.SDK",
+    "Microsoft.Net.Component.4.7.2.TargetingPack",
+    "Microsoft.VisualStudio.Component.Azure.Storage.Emulator",
```
Comment (Pilchie) [inline .vsconfig]: Anyone know where this is used?

File: src/Installers/Windows/.vsconfig [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
+    "Microsoft.Net.Component.4.7.2.SDK",
+    "Microsoft.Net.Component.4.7.2.TargetingPack",
+    "Microsoft.VisualStudio.Component.Azure.Storage.Emulator",
+    "Microsoft.VisualStudio.Component.VC.ATL",
```
Comment (Pilchie) [inline src/Installers/Windows/.vsconfig]: I found the `.sln` files that contained `.vcxproj` files and added the C++ components only to them.

File: .vsconfig (hunk 1 above)
Comment (dougbu) [reply .vsconfig]: Far as I can tell, it isn't. Might have been added in prep for tests using Azure Storage that never materialized or were removed when we dropped Microsoft.AspNetCore.DataProtection.AzureStorage.

File: .vsconfig [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "version": "1.0",
+  "components": [
+    "Microsoft.Net.Component.4.6.1.TargetingPack",
```
Comment (dougbu) [inline .vsconfig]: Do you think we can remove `Microsoft.Net.Component.4.6.2.TargetingPack`, `Microsoft.Net.Component.4.7.TargetingPack`, and `Microsoft.Net.Component.4.7.1.TargetingPack` from the eng/scripts/vs*.json files?

**Review by dougbu — APPROVED** [review]
This may have to do but would prefer we not have so many identical files checked in (if possible). Started an email thread&hellip;

File: .vsconfig (hunk 1 above)
Comment (Pilchie) [reply .vsconfig]: Removed it

File: .vsconfig (hunk 3 above)
Comment (Pilchie) [reply .vsconfig]: Done

Comment (Pilchie) [reply .vsconfig]: Done.

**Pull request merged**